Update project_html_css.md

nnamdi16 opened the pull request:
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

nnamdi16 commented:
Please add my project to the list of submitted projects

Javier-Machin commented:
Hi @nnamdi16! ,

Please keep in mind that you must leave a space between the asterisk and the link.
I fixed it for you and added it. It will show up tomorrow.

nnamdi16 commented:
Thanks

On Oct 6, 2018 01:53, "Javier M" <notifications@github.com> wrote:

> Hi @nnamdi16 <https://github.com/nnamdi16>! ,
>
> Please keep in mind that you must leave a space between the asterisk and
> the link.
> I fixed it for you and added it. It will show up tomorrow.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TheOdinProject/curriculum/pull/8389#issuecomment-427533351>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AVrRItvOgsfGz-JaUewUwca5NZWvOMAiks5uh_73gaJpZM4XLF3V>
> .
>
